internal/fields/validate_test.go

```diff
@@ -528,15 +539,6 @@ func Test_parseElementValue(t *testing.T) {
 				Type: "float",
 			},
 		},
-		// long
-		{
-			key:   "bad long",
-			value: "65537",
-			definition: FieldDefinition{
-				Type: "long",
-			},
-			fail: true,
-		},
```

Comment:
These removed cases are actually allowed by Elasticsearch, so we can probably relax the validation.